apis/v1alpha1/httproute_types.go

```diff
@@ -500,6 +500,16 @@ type HTTPRouteForwardTo struct {
 	// The gateway status for this route should be updated with a
 	// condition that describes the error more specifically.
 	//
+	// The protocol to use is defined using `AppProtocol` field (introduced
```

Comment:
Tiny nit, I think we should avoid backticks here. I haven't seen them used widely in k8s godocs.